pkg/elemental/l10n/en-us.yaml

```diff
@@ -1,10 +1,18 @@
 product:
   elemental: OS Management
+  description: "Elemental is a software stack enabling a centralized, full cloud-native OS management with Kubernetes. <br><br> Cluster Node OSes are built and maintained via container images through the Elemental Toolkit and installed on new hosts using the Elemental CLI. <br><br> The Elemental Operator and the Rancher System Agent enable Rancher Manager to fully control Elemental clusters, from the installation and management of the OS on the Nodes to the provisioning of new K3s or RKE2 clusters in a centralized way."
+  notInstalled: It appears that the Elemental Operator is not installed. To install Elemental Operator please follow the instructions on the official <a target="blank" href="https://rancher.github.io/elemental/elementaloperatorchart-reference/" target="_blank" rel="noopener nofollow">documentation</a>.
```

Comment:
Suggestion, `It appears` is a bit noncommittal for a UI. If we know it's not there we shouldn't sugar coat it (fwiw i and other brits love doing this in other contexts).

I'm not aware of the background for this requirement, but text could be something like....

`The Elemental Operator is required to x,y,x. To install it please follow... `

Reply:
changed. Thanks! I always struggle a bit with the copy 🙇